src/erc20-minter/src/lib.rs

```diff
@@ -1,8 +1,9 @@
 pub mod canister;
+#[cfg(feature = "export-api")]
```

Comment:
in the other bridges we have this inspect module as a submodule of canister

Reply:
Fixed